package.json

```diff
@@ -87,26 +87,26 @@
     "release:major": "standard-version -a --release-as major"
   },
   "devDependencies": {
-    "@commitlint/cli": "17.2.0",
-    "@commitlint/config-conventional": "17.2.0",
+    "@commitlint/cli": "17.4.0",
+    "@commitlint/config-conventional": "17.4.0",
     "@cypress/code-coverage": "3.10.0",
     "@cypress/instrument-cra": "1.4.0",
-    "@trivago/prettier-plugin-sort-imports": "3.4.0",
+    "@trivago/prettier-plugin-sort-imports": "4.0.0",
     "@types/file-saver": "^2.0.5",
-    "@types/i18n": "0.13.5",
+    "@types/i18n": "0.13.6",
     "@types/lodash.countby": "4.6.7",
     "@types/lodash.isequal": "4.5.6",
     "@types/lodash.isobject": "3.0.7",
     "@types/lodash.isstring": "4.0.7",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.43.0",
-    "@typescript-eslint/parser": "5.43.0",
+    "@types/uuid": "9.0.0",
+    "@typescript-eslint/eslint-plugin": "5.48.0",
+    "@typescript-eslint/parser": "5.48.0",
     "concurrently": "7.6.0",
-    "cypress": "11.1.0",
+    "cypress": "12.3.0",
```

Comment:
Similarly, I guess this does not break the cypress tests?

Reply:
No, it is safe to update from 10 to 11 and to 12. Breaking changes occurred when migrating from v9 to v10 (migration guides and code migration tools made the process significantly smoother, well done Cypress 🎉 ). 
Major advantage of using v11 over v10 is the startup speed improvement and v12 added better cookie support and other cool features.